### DISCUSSION TITLE
Donating?

### DISCUSSION THREAD

**Discussion by clo4** [c4spar/cliffy#304] [General]
Thank you for putting so much time into this library, it's a joy to use and the effort is clearly visible in how polished it is.

I don't earn enough money for a monthly sponsorship or Patreon, but if you have something like Ko-fi I'd love to throw a couple bucks your way - and I'm probably not the only one who'd like to!

**Comment by c4spar** [comment]
Hi @SeparateRecords,

thank you so much for your feedback! It's always nice to get feedback like that 🙂

I just set up Github sponsors with monthly and one-time donations, i appreciate any amount 🙂

**Reply by clo4** [reply]
It's not much but hopefully it can buy you a coffee or something :)
Again, thanks for all your work on these OSS projects!

**Reply by c4spar** [reply]
Thank you! @SeparateRecords 🙂
